spec/features/user_can_view_reviews_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+describe "restaurant reviews" do
+  scenario "seen on restaurant show page" do
+    restaurant = Restaurant.create(name: "X",
+                                   address: "1 A Ave",
+                                   phone: "123-456-7890")
+    review_1 = restaurant.reviews.create(name: "Beth",
+                                         review: "It was fine.",
+                                         rating: 5)
+    review_2 = restaurant.reviews.create(name: "Andrew",
+                                         review: "Had better.",
+                                         rating: 3)
+
+    visit restaurant_path(restaurant)
+
+    within("div.reviews div:nth-child(1)") do
+      expect(page).to have_content("Name: Andrew")
```

Comment:
What about using dynamic expected values here that refer to the reviews we just created?